metadata_service/proxy/neo4j_proxy.py

```diff
@@ -81,7 +81,8 @@ def get_table(self, *, table_uri: str) -> Table:
                       watermarks=wmk_results,
                       table_writer=table_writer,
                       last_updated_timestamp=timestamp_value,
-                      source=source)
+                      source=source,
+                      is_view=last_neo4j_record['tbl']['is_view'])
```

Comment:
Would this break if the underlying node attribute does not have `is_view`?

You may use https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/proxy/neo4j_proxy.py#L227

Reply:
Good catch! 👍